src/chrome/content/rules/National_Geographic.com.xml

```diff
@@ -0,0 +1,44 @@
+<!--
+
+	Redirect to HTTP:
+
+		- adventureblog
+		- channel
+		- help
+		- yourshot
+		- yourshotblog
+	
+	Mixed content:
+
+		- archive
+		- intelligenttravel		
+		- news
+		- press
+		- video
+
+-->
+<ruleset name="National Geographic">
+
+	<target host="www.nationalgeographic.com" />
+	<target host="events.nationalgeographic.com" />
+	<target host="genographic.nationalgeographic.com" />
+	<target host="kids.nationalgeographic.com" />
+	<target host="media-members.nationalgeographic.com" />
+		<test url="http://media-members.nationalgeographic.com/static-media/img/favicon.ico" />
+	<target host="media-mmdb.nationalgeographic.com" />
+		<test url="http://media-mmdb.nationalgeographic.com/static-media/img/favicon.ico" />
+	<target host="media-yourshot.nationalgeographic.com" />
+	<target host="members.nationalgeographic.com" />
+	<target host="mmdb.nationalgeographic.com" />
+	<target host="news.nationalgeographic.com" />
+	<target host="press.nationalgeographic.com" />
+	<target host="shop.nationalgeographic.com" />
+
+	<!--	Tracking cookies:
+					-->
+	<securecookie host="^\.nationalgeographic\.com$" name="^(?:gpv_p\d+|s_\w+|__utm)\w$" />
```

Comment:
While we are at it, could you use a [capturing group](https://github.com/EFForg/https-everywhere/blob/master/CONTRIBUTING.md#regex-conventions) in the regex?